blockbase/inc/patterns/footer-with-site-title.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Footer with Site Title
+ *
+ * @package Blockbase
+ */
+
+return array(
+	'title'      => __( 'Footer with site title', 'blockbase' ),
+	'categories' => array( 'blockbase' ),
+	'blockTypes' => array( 'core/template-part/footer' ),
+	'content'    => '<!-- wp:group {"layout":{"inherit":true}} -->
+	<div class="wp-block-group">
+		<!-- wp:group {"layout":{"type":"flex"},"style":{"spacing":{"padding":{"top":"80px","bottom":"30px"}}}} -->
```

Comment:
I noticed that the items in the footer are not centered, should they be?

<img width="1904" alt="Screen Shot 2022-01-17 at 11 21 38 AM" src="https://user-images.githubusercontent.com/5375500/149805751-7612949c-681f-47a6-9764-45747667230b.png">

Reply:
Good call, done in 53ea5310e